src/Cargo/sdk/Sdk.props

```diff
@@ -15,24 +16,30 @@
     <ManagedLanguageTargetsGotImported Condition="'$(MSBuildProjectExtension)' == '.fsproj'">true</ManagedLanguageTargetsGotImported>
   </PropertyGroup>
 
-  <Import Project="$(CustomBeforeCargoProps)" Condition=" '$(CustomBeforeCargoProps)' != '' And Exists('$(CustomBeforeCargoProps)') " />
+  <Import Project="$(CustomBeforeCargoProps)"
+    Condition=" '$(CustomBeforeCargoProps)' != '' And Exists('$(CustomBeforeCargoProps)') " />
 
   <PropertyGroup>
     <!-- Disable default Compile and EmbeddedResource items for Cargo projects -->
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
-    
+
     <!--
-      NuGet should always restore Cargo projects with "PackageReference" style restore.  Setting this property will cause the right thing to happen even if there aren't any PackageReference items in the project.
+      NuGet should always restore Cargo projects with "PackageReference" style restore.  Setting this
+    property will cause the right thing to happen even if there aren't any PackageReference items in
+    the project.
     -->
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
 
     <!-- Targeting packs shouldn't be referenced as Cargo projects don't compile .net code. -->
-    <DisableImplicitFrameworkReferences Condition="'$(DisableImplicitFrameworkReferences)' == ''">true</DisableImplicitFrameworkReferences>
```

Comment:
I don't know about these. This is changing the value of the property. I'm pretty sure that MSBuild trims this, but the whitespace will be included in the raw value.